simplified-accounts-json/src/main/java/org/nypl/simplified/accounts/json/AccountAuthenticationCredentialsJSON.kt

```diff
@@ -49,7 +55,7 @@ object AccountAuthenticationCredentialsJSON {
     authObject.put("@version", this.currentSupportedVersion)
     authObject.put("authenticationDescription", credentials.authenticationDescription)
 
-    when (credentials) {
+    val ignored = when (credentials) {
```

Comment:
This is totally fine with me but I think we should consider adopting the `@Exhaustive` annotation in the near future:

https://github.com/cashapp/exhaustive

I have a feeling something like this will eventually be adopted by the language itself. Good discussion of the alternatives here:

https://github.com/cashapp/exhaustive#alternatives-considered